src/game_inventory.cpp

```diff
@@ -528,13 +528,19 @@ class comestible_inventory_preset : public inventory_selector_preset
             }, _( "CONSUME TIME" ) );
 
             append_cell( [this, &player_character]( const item_location & loc ) {
+                std::string sealed = "";
+                if( loc.has_parent() ) {
+                    sealed = loc.parent_item()->contents.get_sealed_summary() ==
+                             item_contents::sealed_summary::all_sealed ? "sealed" : "";
```

Comment:
```suggestion
                             item_contents::sealed_summary::all_sealed ? _( "sealed" ) : "";
```

Reply:
1. I suppose it could be improved to check the exact pocket holding the food item. It would be more robust.
2. That is a good suggestion, and I agree another PR would be the place to address that.